sdk/typescript/src/types/sui-bcs.ts

```diff
@@ -16,12 +15,12 @@ bcs
   .registerType(
     'utf8string',
     (writer, str) => {
-      let bytes = Array.from(Buffer.from(str));
+      const bytes = Array.from(new TextEncoder().encode(str));
```

Comment:
@666lcz It doesn't looks like there's tests covering this, so if you could help me double-check that this behavior is the same that'd be great!

Reply:
This should be covered by https://github.com/MystenLabs/sui/blob/4180f238c9a7924e492b09536cb9c51d02641071/sdk/typescript/test/e2e/entry-point-string.test.ts#L56